.github/workflows/e2e-manual-icad.yaml

```diff
@@ -25,7 +25,7 @@ on:
         default: master
         options:
           - master
-          - v0.4.1
+          - v0.4.2
```

Comment:
I will create this tag in interchain-accounts-demo repo with v6.0.0.

Reply:
Is the tag just for demo purposes? We shouldn't be testing the demo repo with v6?